tests/unit/App.spec.ts

```diff
@@ -14,14 +12,11 @@ import {
 import SbcHeader from 'sbc-common-components/src/components/SbcHeader.vue'
 import SbcFooter from 'sbc-common-components/src/components/SbcFooter.vue'
 // import mockRouter from './MockRouter'
+import { getVuexStore } from '@/store/'
 
 const vuetify = new Vuetify({})
 
-const store = new Vuex.Store<any>({
-  getters: {
-    isMobile (): any {}
-  }
-})
+const store = getVuexStore()
```

Comment:
This is the fix for "isRoleStaff is not defined" and other getter/action test issues.